src/apps/config/leader.lua

```diff
@@ -70,16 +69,31 @@ end
 
 function Leader:set_initial_configuration (configuration)
    self.current_configuration = configuration
-   self.current_app_graph = self.setup_fn(configuration)
    self.current_in_place_dependencies = {}
-   self.current_in_place_dependencies =
+
+   -- Start the followers and configure them.
+   local follower_configurations = self.setup_fn(configuration)
+
+   -- Calculate the dependences
+   self.in_place_dependencies =
       self.support.update_mutable_objects_embedded_in_app_initargs (
-         {}, self.current_app_graph, self.schema_name, 'load', '/',
-         self.current_configuration)
-   local initial_app_graph = app_graph.new() -- Empty.
-   local actions = self.support.compute_config_actions(
-      initial_app_graph, self.current_app_graph, {}, 'load')
-   self:enqueue_config_actions(actions)
+            app_graph.new(), follower_configurations, self.schema_name, 'load',
```

Comment:
This doesn't look right to me; the default `update_mutable_objects_embedded_in_app_initargs` function goes like this:

```lua
   update_mutable_objects_embedded_in_app_initargs = function(
         in_place_dependencies, app_graph, schema_name, verb, path, arg)
      return compute_mutable_objects_embedded_in_app_initargs(app_graph)
   end,
```

Obviously this is generic fallback, but still.  I think the first argument should probably still be the initial `in_place_dependencies` which is the empty map `{}`, and then we need to replace the `app_graph` argument with a table of app graphs (`follower_configurations` or `worker_app_graphs` or whatever that ends up being).  Of course then we have to adapt the implementation to expect a table of app graphs, but I think you've done that below.  WDYT?